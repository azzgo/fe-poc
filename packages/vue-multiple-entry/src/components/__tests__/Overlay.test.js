import Ovlerlay from '../Overlay.vue'
import { shallowMount } from '@vue/test-utils'

test('Ovlerlay should work as expect', () => {
  const comp = shallowMount(Ovlerlay, {
    propsData: {
      visible: false,
    },
  })
  comp.setProps({
    visible: true,
  })
  expect(document.body.innerHTML).toMatchSnapshot()

  comp.setProps({
    visible: false,
  })

  expect(document.body.innerHTML).toMatchSnapshot()
})
