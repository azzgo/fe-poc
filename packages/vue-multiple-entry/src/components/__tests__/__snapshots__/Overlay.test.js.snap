// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ovlerlay should work as expect 1`] = `
"
<div id=\\"loading\\"></div>
<div class=\\"overlay\\"><div class=\\"overlay-mask\\"></div> <div class=\\"overlay-container\\"></div></div>"
`;

exports[`Ovlerlay should work as expect 2`] = `
"
<div id=\\"loading\\"></div>
<!---->"
`;
